src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,16 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform $transition-time $transition-linear,
+      color $transition-time $transition-linear;
```

Comment:
```suggestion
    transition: 
      transform $transition-time $transition-linear,
      color $transition-time $transition-linear;
```

Reply:
If I try to move transform property to next line, I get lint mistake
![image](https://user-images.githubusercontent.com/57047930/200349601-fb17af23-0f59-4ba8-9034-71b67cf74870.png)